packages/conform-dom/formdata.ts

```diff
@@ -148,9 +146,7 @@ export function getValue(target: unknown, name: string): unknown {
 /**
  * Check if the value is a plain object
  */
-export function isPlainObject(
-	obj: unknown,
-): obj is Record<string | number | symbol, unknown> {
+export function isPlainObject(obj: unknown): obj is Record<keyof any, unknown> {
```

Comment:
@edmundhung also simplified object types to use `keyof any` which resolves to the same type.